lib/core/src/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobin.hs

```diff
@@ -871,7 +871,7 @@ runSelection limit mExtraCoinSource available minimumBalance =
   where
     initialState :: SelectionState
     initialState = SelectionState
-        { selected = UTxOIndex.empty
+        { selected = mempty
```

Comment:
Similarly to my earlier comment, I think the use of `mempty` here makes the code harder to read, especially in a web-browser.

If someone is using an editor with type feedback, then of course, they can look it up, but it's an extra step that isn't currently necessary.